juju/juju
# fix: modelmanager mock tests for model creation

Change the tests to have a non nil model service, this means the mocks calls now need to be expected for model create. Before, if the model service was nil the model was not created in DQLite so this didn't need to be tested.

This change is needed as part of #18003 




## Checklist




## QA steps
```
go test -v -tags=libsqlite3,dqlite ./apiserver/facades/client/modelmanager/... -check.v -check.f Suite
```

https://warthogs.atlassian.net/browse/JUJU-6626
